.github/actions/publish_to_pages_production/action.yml

```
@@ -0,0 +1,21 @@
+name: publish_to_pages_production
+inputs:
+  CLOUDFLARE_API_TOKEN:
+    description: 'Cloudflare Pages token'
+    required: true
+  CLOUDFLARE_ACCOUNT_ID:
+    description: 'Cloudflare Account ID'
+    required: true
+runs:
+  using: composite
+  env:
+    CLOUDFLARE_API_TOKEN: ${{ inputs.CLOUDFLARE_API_TOKEN }}
+    CLOUDFLARE_ACCOUNT_ID: ${{ inputs.CLOUDFLARE_ACCOUNT_ID }}
+  steps:
+  - name: Publish to cloudflare pages (production)
+    environment: Production
+    run: |-
+      cd public
+      npx wrangler pages publish . --project-name=deriv-static-content-pages --branch=main
```

```suggestion
      npx wrangler pages deploy . --project-name=deriv-static-content-pages --branch=main
```

---

good catch @habib-deriv 
@amir-deriv I think we better to use a specific version of wrangler too.
You can simply install it before using it. 
```suggestion
      npm i wrangler@3.1.0
      npx wrangler pages deploy . --project-name=deriv-static-content-pages --branch=main
```